src/components/ParsedHtml/style.css

```diff
@@ -0,0 +1,8 @@
+.parsed-link {
+    color: #7F99FB !important;
```

Comment:
@reallybeard do you have any suggested ways to make this work with dark/light?

Reply:
You can do
```
body.chakra-ui-dark .parsed-link {}
```

Reply:
@reallybeard @0xdef1cafe done, I went with explicitely setting color for dark and light themes, lint does not like empty ruleset for css